deps/NativePath/NativePath.h

```diff
@@ -3021,15 +3021,15 @@ typedef uint32_t uint4 __attribute__((ext_vector_type(4)));
 #endif
 
 #if !__has_builtin(__builtin___strlcat_chk)
-	#error \"__strlcat_chk clang built-in not available\"
+	//#error \"__strlcat_chk clang built-in not available\"
```

Comment:
This change can be added directly in a separate PR

Reply:
I think it should be dealt separately. I can add a `// FIXME xplat-editor` like I did in other places.